web/client/components/map/leaflet/MeasurementSupport.jsx

```diff
@@ -2,21 +2,217 @@ const PropTypes = require('prop-types');
 const React = require('react');
 const assign = require('object-assign');
 var L = require('leaflet');
-const {slice} = require('lodash');
-const {reproject, calculateAzimuth, calculateDistance, transformLineToArcs} = require('../../../utils/CoordinatesUtils');
+const {
+    slice
+} = require('lodash');
+const {
+    reproject,
+    calculateAzimuth,
+    calculateDistance,
+    transformLineToArcs
+} = require('../../../utils/CoordinatesUtils');
+const {
+    sqmTosqft,
```

Comment:
Can't we translate all these function into one (e.g. convertUoM(value, source, dest) ?

Reply:
there are already getFormattedLength and getFormattedArea which takes source by default in meters and square meters

Reply:
those are for formatting, I mean a function for converting between uoms